api/namex/resources/payment/payment.py

```diff
@@ -124,6 +125,25 @@ def handle_auth_error(ex):
     return response
 
 
+def custom_strftime_suffix(d):
+    return 'th' if 11 <= d <= 13 else {1: 'st', 2: 'nd', 3: 'rd'}.get(d % 10, 'th')
```

Comment:
If I understand correctly, this returns "1st", "2nd", "3rd" or "Xth".

What does this do for 11, 12, 13?